pages/microservices_architecture.md

```diff
@@ -245,13 +246,13 @@ To get started with developping applications that rely on a Consul registry, you
+The K/V store can be modified using either it's UI available at [http://localhost:8500/v1/kv/](http://localhost:8500/v1/kv/) or it's [REST API](https://www.consul.io/intro/getting-started/kv.html). However changes made this way are temporary and will be lost on Consul server/cluster shutdown. So, in order to help you interact easily with the Key/Value store and manage your configuration as simple YAML files, the JHipster Team has developed a small tool: the [consul-config-loader](https://github.com/jhipster/consul-config-loader). The **consul-config-loader** is automatically configured when starting Consul from the `consul.yml` docker-compose file but it can also be run as a standalone tool.
 It can be run in two modes:
 
-- a **dev** mode, where YAML files from the `central-server-config` directory are automatically loaded into Consul. Moreover any change to this directory will be immediately synchronized with the Key/Value store.
-- a **prod** mode, that uses Git2Consul to setup the YAML files contained in a git repository as configuration source for the Key/Value store.
+- a **dev** mode, where YAML files from the `central-server-config` directory are automatically loaded into Consul. Moreover any change to this directory will be immediately synchronized with the K/V store.
+- a **prod** mode, that uses Git2Consul to setup the YAML files contained in a git repository as configuration source for the K/V store.
 
-Note that as with the JHipster Registry, your configuration files will need to be named `appname-profile.yml` where appname and profile correspond to the application’s name and profile of the service that you want to configure. For example, adding properties in a `consulapp-prod.yml` file will set those properties only for the application named `consulapp` started with a `prod` profile. Moreover, properties defined in `application.yml` will be set for all your applications.
+Note that as with the JHipster-Registry, your configuration files will need to be named `appname-profile.yml` where appname and profile correspond to the application’s name and profile of the service that you want to configure. For example, adding properties in a `consulapp-prod.yml` file will set those properties only for the application named consulapp started with a prod profile. Moreover, properties defined in `application.yml` will be set for all your applications.
```

Comment:
We normally don't have a "-" in JHipster Registry, and I liked to have the backquotes in the appname and profile, that's how it's usually done in the rest of the documentation